_plugins/api.rb

```diff
@@ -54,7 +54,7 @@ def self.generate_team_endpoint(site)
       return if team.empty?
       fields = ['first_name', 'last_name', 'full_name', 'image',
          'pif-round', 'bio', 'email', 'slack', 'location',
-         'languages', 'technologies', 'specialties']
+         'languages', 'technologies', 'specialties', 'hidden']
```

Comment:
We don't have to do it now, but we can deprecate the `hidden` key whenever we want to. The front page doesn't depend on it since it now uses the existence of an image to determine whether to display the name + photo. (See: [the template code](https://github.com/18F/18f.gsa.gov/blob/staging/pages/index.html#L96) and [the `with_pic` filter](https://github.com/18F/18f.gsa.gov/blob/staging/_plugins/author.rb#L23-L33) for how we implemented that.